packages/schema/src/config/app.ts

```diff
@@ -189,12 +189,13 @@ export default defineUntypedSchema({
   },
 
   /**
-   * A path to an HTML file, the contents of which will be inserted into any HTML page
+   * Boolean or a path to an HTML file with the contents of which will be inserted into any HTML page
    * rendered with `ssr: false`.
    *
-   * By default Nuxt will look in `~/app/spa-loading-template.html` for this file.
+   * Default value: `false` (Disabled).
    *
-   * You can set this to `false` to disable any loading indicator.
+   * If true, Nuxt will look for `~/app/spa-loading-template.html` file, if it does not exist,
```

Comment:
I think the presence of `~/app/spa-loading-template.html` should enable the template. I think the default Nuxt image should only be used if this is set to `true` and if that file is not present:

- `true` +  `~/app/spa-loading-template.html` - template provided
- `true` - default nuxt icon
- `false` - no template
- unset +  `~/app/spa-loading-template.html` - template provided
- unset - no template

What do you think?

Reply:
Thanks for your review.
I implemented in that way.

Since its my first contribution, I expect change request.

Reply:
nice work